data/json/faults.json

```diff
@@ -36,17 +39,43 @@
     "type": "fault",
     "name": "diesel pump",
     "description": "Required to pump and pressurise diesel from a vehicles tank.",
+    "time": 240000,
     "skills": [ [ "mechanics", 4 ] ],
     "requirements": {
       "qualities": [ { "id": "WRENCH_FINE", "level": 1 } ],
       "components": [ [ [ "pump_complex", 1 ] ] ]
     }
   },
   {
+    "id": "fault_engine_filter_air",
+    "type": "fault",
+    "name": "air filter",
+    "description": "An expired filter reduces fuel efficiency and increases smoke production.",
+    "time": 30000,
+    "skills": [ [ "mechanics", 1 ] ],
+    "requirements": {
+      "qualities": [ { "id": "SCREW", "level": 1 } ],
+      "components": [ [ [ "filter", 1 ], [ "filter_makeshift", 1 ] ] ]
```

Comment:
Hmmn, I was trying to represent that the material both are comprised of is analogous but I suppose we do need differing types. Perhaps the improvised repairs manual should allow you to improvise one type from the other?